CommonWallet/Core/Classes/Modules/Transfer/ViewModel/AmountInputViewModel.swift

```diff
@@ -109,4 +113,8 @@ public final class AmountInputViewModel: AmountInputViewModelProtocol, MoneyPres
 
         return false
     }
+
+    public func didUpdateAmount(to newAmount: String) {
```

Comment:
Better to pass Decimal and apply formatter & limit as in init. Otherwise we potential can have invalid amount